src/Sql/dbo/Views/SsoConfigView.sql

```diff
@@ -1,6 +1,8 @@
 ﻿CREATE VIEW [dbo].[SsoConfigView]
 AS
 SELECT
-    *
+    SSO.*
```

Comment:
Please also include `O.[Identifier]` in the view output, something tells me we'll probably need it somewhere in the C# pipeline/logic tied to these records.

Reply:
Updated logic and moved to sproc